### PR TITLE
Snare-Tanner Communicaton

### PR DESCRIPTION
**Pull request by rnehra01**
Fix https://github.com/mushorg/tanner/issues/100

### CODE REVIEW

File: snare.py [hunk 1]
```diff
@@ -235,6 +214,37 @@ def handle_request(self, request, payload):
...
+            else:
+                content_type = mimetypes.guess_type(payload_content['value'])[0]
+                content = payload_content['value'].encode('utf-8')
+
+            if 'headers' in payload_content:
+                for name, val in payload_content['headers']:
+                    response.add_header(name, val)
+        else:
+            response = aiohttp.Response(
+                self.writer, status=payload_content['status_code'], http_version=request.version
```
Comment (afeena) [inline snare.py]: I think if we have some problem in tanner, we should keep it private and show e.g. index page for the user, how do you think?

File: snare.py [hunk 2]
```diff
@@ -192,30 +192,9 @@ def handle_request(self, request, payload):
...
-                soup.body.append(script_tag)
-                content = str(soup).encode()
-
-            else:
-                content_type = mimetypes.guess_type(payload_content)[0]
-                content = payload_content.encode('utf-8')
-        else:
+       
+        detection = event_result['response']['message']['detection']
+        if detection['type'] == 1:
```
Comment (afeena) [inline snare.py]: Maybe it is suitable to make separate function for parsing tanner response?

File: snare.py [hunk 3]
```diff
@@ -257,6 +216,60 @@ def handle_request(self, request, payload):
...
+                soup.body.append(script_tag)
+                content = str(soup).encode()
+            else:
+                content_type = mimetypes.guess_type(payload_content['value'])[0]
+                content = payload_content['value'].encode('utf-8')
+
+            if 'headers' in payload_content:
+                headers =  payload_content['headers']
+        else:
+            status_code = payload_content['status_code']
```
Comment (afeena) [inline snare.py]: If we have e.g. 500 error on tanner side, why do we need to pass it to the end user? Maybe it's better to return current page?

Comment (afeena) [inline snare.py]: Ok, my bad :)
But what about tanner errors?

**Comment by afeena** [comment]
I think we can merge it in develop :) I want to test it with tanner a bit

File: snare.py (hunk 3 above)
Comment (rnehra01) [inline snare.py]: Sure, I'll add that too.

**Pull request merged**